CPS-????/README.md

```diff
@@ -0,0 +1,137 @@
+---
+CPS: 24
+Title: Canonical CBOR Serialization Standard
```

Comment:
```suggestion
Title: Canonical CBOR Serialization
```
If we had a "Standard" for how to deal with this problem, we wouldn't need to make a Problem Statement about it. 😆  The word might be applicable in CIPs (though rare, since most CIPs _aren't_ standards, since generally merged when not yet `Active`), but even then perhaps not... if the Ledger & different realms of software end up treating CBOR standardisation differently.